test/conformance/ptp/ptp.go

```diff
@@ -943,3 +980,96 @@ func ptpEventEnabled() bool {
 	}
 	return ptpConfig.Spec.EventConfig.EnableEventPublisher
 }
+
+func getNICInfo(pod v1core.Pod) map[string]string {
+
+	var ptpSupportedInterfaces []string = getPtpMasterSlaveAttachedInterfaces(pod)
+	var stdout bytes.Buffer
+
+	var ptpInterfaceNicMapping = make(map[string]string)
+
+	for _, interf := range ptpSupportedInterfaces {
+		PCIAddr := ""
+		var err error
+
+		Eventually(func() error {
+			stdout, err = pods.ExecCommand(client.Client, pod, PtpContainerName, []string{"readlink", "-f", fmt.Sprintf("/sys/class/net/%s", interf)})
```

Comment:
Why this over using ethtool?

Reply:
@josephdrichard No sure who took this decision, it is being used [in the existing code](https://github.com/openshift/ptp-operator/blob/master/test/conformance/ptp/ptp.go#L776), which is reused here. Maybe @aneeshkp knows?

Reply:
Yes, using existing readlink method, to read pci information .